test/Serilog.Tests/Core/ForContextExtensionsTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Serilog.Events;
+using Serilog.Tests.Support;
+using Xunit;
+
+namespace Serilog.Tests.Core
+{
+    public class ForContextExtensionsTests
+    {
+        [Theory]
+        [InlineData(LogEventLevel.Debug, LogEventLevel.Debug)]
+        [InlineData(LogEventLevel.Debug, LogEventLevel.Information)]
+        [InlineData(LogEventLevel.Debug, LogEventLevel.Error)]
+        [InlineData(LogEventLevel.Debug, LogEventLevel.Fatal)]
+        [InlineData(LogEventLevel.Debug, LogEventLevel.Warning)]
+        public void ShouldEnrichLogEventWhenLevelIsSameOrHigherThanMinLevel(LogEventLevel logMinLevel, LogEventLevel propertLogLevel)
+        {
+            // Arrange
```

Comment:
I know there are other tests in the codebase commented with `// Arrange` etc., but we're not consistent - I'd prefer we drop these as after a time they become boilerplate.

Reply:
Roger that - I'm renaming the test class name to reflect the name change of the extension method class.